src/views/InvolvementsAll/index.jsx

```diff
@@ -18,7 +18,7 @@ import membershipService, { NonGuestParticipations } from 'services/membership';
 import sessionService from 'services/session';
 import InvolvementsGrid from './components/InvolvementsGrid';
 import Requests from './components/Requests';
-import styles from './InvolvementsAll.module.css';
+import styles from './Involvements.module.css';
```

Comment:
If we aren't using InvolvementsAll.module.scss anymore, that file should be removed.